webapp/webapp.spec.yaml

```diff
@@ -1048,3 +1068,46 @@ components:
           items:
             type: string
             example: CVE-2016-0800
+
+    RpmsRequest:
+      type: object
+      properties:
+        srpm_name_list:
+          type: array
+          items:
+            type: string
+            example: openssl
+        rpm_name_list:
+          type: array
+          items:
+            type: string
+            example: openssl
+        content_set_list:
+          type: array
+          items:
+            type: string
+            example: rhel-7-desktop-rpms
+
+    RpmsResponse:
+        properties:
+          last_change:
+            example: '2020-04-16 20:07:58.500192+00'
+          srpm_name_list:
+            type: object
+            properties:
+              openssl:
```

Comment:
Is this typo or is it going to work to have ```openssl``` property? Probably the syntax should be similar to this?
```
srpm_name_list:
          type: object
          additionalProperties:
              type: object
              additionalProperties:
                  type: array
                  items:
                    type: string
                    example: 'openssl-libs'
```